<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=generator content="Wowchemy 5.6.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap" media=print onload='this.media="all"'><meta name=author content="Martin Hölzer"><meta name=description content="Personal page"><link rel=alternate hreflang=en-us href=https://hoelzer.github.io/><meta name=theme-color content="#1565c0"><link rel=stylesheet href=./css/vendor-bundle.min.c7b8d9abd591ba2253ea42747e3ac3f5.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=./css/wowchemy.7d31d7a85d84d3fa1c19b09946b72266.css><link rel=stylesheet href=./css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=./css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><script src=https://identity.netlify.com/v1/netlify-identity-widget.js></script><link rel=alternate href=./index.xml type=application/rss+xml title="Martin Hölzer"><link rel=manifest href=./manifest.webmanifest><link rel=icon type=image/png href=./media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=./media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://hoelzer.github.io/><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@martinhoelzer"><meta property="twitter:creator" content="@martinhoelzer"><meta property="og:site_name" content="Martin Hölzer"><meta property="og:url" content="https://hoelzer.github.io/"><meta property="og:title" content="Martin Hölzer"><meta property="og:description" content="Personal page"><meta property="og:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="twitter:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2030-06-01T13:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://hoelzer.github.io/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://hoelzer.github.io/"}</script><title>Martin Hölzer</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main class=page-wrapper><script src=./js/wowchemy-init.min.613040fe4f2c0f007b4dcb64404201cb.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><header class=header--fixed><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=./><img src=./media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=./><img src=./media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=./#about data-target=#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=./#publications data-target=#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=./#experience data-target=#experience><span>Experience</span></a></li><li class=nav-item><a class=nav-link href=./#contact data-target=#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://github.com/hoelzer target=_blank rel=noopener aria-label=github><i class="fab fa-github" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span>
</a><a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span>
</a><a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><span class="js-widget-page d-none"></span><section id=about class="home-section wg-about"><div class=home-section-bg></div><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class="avatar avatar-circle" width=270 height=270 src=./authors/admin/avatar_hu5efe4ffaac1945d607bbe257eff5fff6_208945_270x270_fill_q75_lanczos_center.jpg alt="Martin Hölzer"><div class=portrait-title><h2>Martin Hölzer</h2><h3>Bioinformatics Researcher, Team Lead @RKI</h3><h3><a href=https://www.rki.de/EN/Home/homepage_node.html target=_blank rel=noopener><span>Robert Koch Institute, Berlin, Germany</span></a></h3></div><ul class=network-icon aria-hidden=true><li><a href=./#contact aria-label=envelope><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://twitter.com/martinhoelzer target=_blank rel=noopener aria-label=twitter><i class="fab fa-twitter big-icon"></i></a></li><li><a href="https://scholar.google.de/citations?user=DMZ7Hc8AAAAJ&amp;hl=en" target=_blank rel=noopener aria-label=google-scholar><i class="ai ai-google-scholar big-icon"></i></a></li><li><a href=https://orcid.org/0000-0001-7090-8717 target=_blank rel=noopener aria-label=orcid><i class="fab fa-orcid big-icon"></i></a></li><li><a href=https://github.com/hoelzer target=_blank rel=noopener aria-label=github><i class="fab fa-github big-icon"></i></a></li><li><a href=./files/resume/cv-martin-hoelzer.pdf aria-label=cv><i class="ai ai-cv big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-1"></div><div class="col-12 col-lg-7"><div class=vertical-center , align=justify><div class=article-style><div class=main-content><p><strong>About me</strong></p><p>I am a researcher and team lead at the <a href=https://www.rki.de/EN/Content/Institute/DepartmentsUnits/MF/MF1/mf1_node.html target=_blank rel=noopener>Genome Competence Center</a> of the <a href=https://www.rki.de/EN/Home/homepage_node.html target=_blank rel=noopener>Robert Koch Institute</a> in Berlin, Germany. We study infectious diseases in a One Health context by applying real-time nanopore sequencing, analyzing environmental samples such as wastewater, and studying microbial evolution using pangenomics and AI for genomic surveillance.</p><p>I studied Bioinformatics at the University of Jena, Germany (FSU) and achieved my PhD by specializing in <a href=https://www.rna.uni-jena.de/ target=_blank rel=noopener>RNA Bioinformatics and High-Throughput Analysis</a>. Following this, I moved to the European Bioinformatics Institute (EMBL-EBI), Hinxton (UK) to work in <a href=https://www.ebi.ac.uk/about/teams/microbiome-informatics/ target=_blank rel=noopener>Microbiome Informatics</a>. Since 2020, I have continued my academic journey at the RKI in Berlin, the national institute for public health in Germany, conducting research with my team and (inter)national colleagues in the following areas:</p><p><strong>Research interests</strong></p><ul><li>Microbial bioinformatics</li><li>Genomic surveillance</li><li>Pathogen detection and characterization</li><li>Nanopore real-time sequencing</li><li>Transcriptomics and gene expression</li><li>Metagenomics and metatranscriptomics</li><li>Cloud computing, automation, and reproducibility with <a href=https://nextflow.io target=_blank rel=noopener>Nextflow</a></li></ul></div></div></div></div></div></div></section><section id=publications class="home-section wg-pages"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Selected publications</h1><p class=mt-1>For a complete list, see my <a href="https://scholar.google.de/citations?user=DMZ7Hc8AAAAJ&amp;hl=en" target=_blank rel=noopener>Google Scholar</a>.</p></div><div class="col-12 col-lg-8"><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Eva Aßmann</span>, <span>Shelesh Agrawal</span>, <span>Laura Orschler</span>, <span>Sindy Böttcher</span>, <span>Susanne Lackner</span>, <span><strong>Martin Hölzer</strong></span>
</span>(2024).
<a href=./publication/2024-impact-of-reference-design.html>Impact of reference design on estimating SARS-CoV-2 lineage abundances from wastewater sequencing data</a>.
GigaScience.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/gigascience/giae051 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/rki-mf1/vlq-nf target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/upbqj/ target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Kevin Lamkiewicz</span>, <span>Lisa-Marie Barf</span>, <span>Konrad Sachse</span>, <span><strong>Martin Hölzer</strong></span>
</span>(2024).
<a href=./publication/2024-ribap.html>RIBAP: a comprehensive bacterial core genome annotation pipeline for pangenome calculation beyond the species level</a>.
Genome Biology, 25 (1), 170.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03312-9 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/hoelzer-lab/ribap target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/g52rb target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><strong>Martin Hölzer</strong></span>
</span>(2024).
<a href=./publication/2024-pocp-nf.html>POCP-nf: an automatic Nextflow pipeline for calculating the percentage of conserved proteins in bacterial taxonomy</a>.
Bioinformatics, 40 (4), btae175.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/bioinformatics/btae175 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/hoelzer/pocp target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/2tzd9 target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Guillermo Rangel-Pineros, Alexandre Almeida, Martin Beracochea, Ekaterina Sakharova, Manja Marz, Alejandro Reyes Muñoz, <strong>Martin Hölzer</strong></span>, <span>Robert D Finn</span>
</span>(2023).
<a href=./publication/2023-virify.html>VIRify: an integrated detection, annotation and taxonomic classification pipeline using virus-specific protein profile hidden Markov models</a>.
PLOS Computational Biology, 19 (8), e1011422.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1371/journal.pcbi.1011422 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/EBI-Metagenomics/emg-viral-pipeline target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.17605/OSF.IO/FBRXY target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Sara Saheb Kashaf</span>, <span>Diana M Proctor</span>, <span>Clay Deming</span>, <span>Paul Saary</span>, <span><strong>Martin Hölzer</strong></span>, <span>Monica E Taylor</span>, <span>Heidi H Kong</span>, <span>Julia A Segre</span>, <span>Alexandre Almeida</span>, <span>Robert D Finn</span>
</span>(2021).
<a href=./publication/2022-skin-microbiome.html>Integrating cultivation and metagenomics for a multi-kingdom view of skin microbiome diversity and functions</a>.
Nature Microbiology, 7 (1), 169-179.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.nature.com/articles/s41564-021-01011-w target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/skinmicrobiome/skin_MAGs/releases/tag/v1.0 target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=http://ftp.ebi.ac.uk/pub/databases/metagenomics/genome_sets/skin_microbiome target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Metagenome assemblies</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Christian Brandt</span>, <span>Sebastian Krautwurst</span>, <span>Riccardo Spott</span>, <span>Mara Lohde</span>, <span>Mateusz Jundzill</span>, <span>Mike Marquet</span>, <span><strong>Martin Hölzer</strong></span>
</span>(2021).
<a href=./publication/2021-porecov.html>poreCov - an easy to use, fast, and robust workflow for SARS-CoV-2 genome reconstruction via nanopore sequencing</a>.
Frontiers in Genetics, 12, 711437.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.3389/fgene.2021.711437 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/replikation/poreCov target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span>Will A Overholt</span>, <span><strong>Martin Hölzer</strong></span>, <span>Patricia Geesink</span>, <span>Celia Diezel</span>, <span>Manja Marz</span>, <span>Kirsten Küsel</span>
</span>(2020).
<a href=./publication/2020-aquadiva-long-reads.html>Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome‐assembled genomes from a complex aquifer system</a>.
Environmental microbiology, 22(9), 4000-4013.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1111/1462-2920.15186 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a></p></div><div class="pub-list-item view-citation" style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><strong>Martin Hölzer</strong></span>, <span>Manja Marz</span>
</span>(2019).
<a href=./publication/2019-transcriptome-assembly.html>De novo transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers</a>.
GigaScience, 8 (5), giz039.<p><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/gigascience/giz039 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.rna.uni-jena.de/supplements/assembly/index.html target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Electronic Supplement</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.17605/OSF.IO/5ZDX4 target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></p></div></div></div></div></section><section id=experience class="home-section wg-experience"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Experience</h1><p class=mt-1>See <a href=files/resume/cv-martin-hoelzer.pdf>my full CV</a> for details.</p></div><div class="col-12 col-lg-8"><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border exp-fill">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.rki.de/EN/Home/homepage_node.html target=_blank rel=noopener><img src=./media/icons/brands/rki.svg width=56px height=56px alt="Bioinformatics & Translational Research, Genome Competence Center, RKI" loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">Researcher & Team Lead</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.rki.de/EN/Home/homepage_node.html target=_blank rel=noopener>Bioinformatics & Translational Research, Genome Competence Center, RKI</a></div><div class="text-muted exp-meta">Aug 2020 –
Present
<span class=middot-divider></span>
<span>Berlin, Germany</span></div></div></div><div class=card-text>Responsible for bioinformatics research development focusing on pandemic prepardness, One Health, and pathogen evolution. Leading an interdisciplinary team conducting research and method development to characterize infectious disease agents by modern sequencing.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.ebi.ac.uk/about/teams/microbiome-informatics target=_blank rel=noopener><img src=./media/icons/brands/ebi.svg width=56px height=56px alt="Microbiome Informatics, European Bioinformatics Institute (EMBL-EBI), UK" loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">Visiting Researcher</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.ebi.ac.uk/about/teams/microbiome-informatics target=_blank rel=noopener>Microbiome Informatics, European Bioinformatics Institute (EMBL-EBI), UK</a></div><div class="text-muted exp-meta">Oct 2019 –
Mar 2020
<span class=middot-divider></span>
<span>Hinxton, United Kingdom</span></div></div></div><div class=card-text>Working in the research group of Dr. Rob Finn on the analysis and classification of metagenomic samples using Nanopore long-read sequencing data and hybrid approaches. Contributor to the VIRify virus detection pipeline.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.rna.uni-jena.de/ target=_blank rel=noopener><img src=./media/icons/brands/lauchlachs-vertical.svg width=56px height=56px alt="RNA Bioinformatics & High-Throughput Analysis, University of Jena" loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">Post-doc & Team Lead</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.rna.uni-jena.de/ target=_blank rel=noopener>RNA Bioinformatics & High-Throughput Analysis, University of Jena</a></div><div class="text-muted exp-meta">Jan 2018 –
Jul 2020
<span class=middot-divider></span>
<span>Jena, Germany</span></div></div></div><div class=card-text>Team leader of an associated bioinformatics group at the Marz lab. Research on novel applications for emerging Nanopore long-read sequencing data in virus bioinformatics and metagenomics.</div></div></div></div></div><div class="row experience"><div class="col-auto text-center flex-column d-none d-sm-flex"><div class="row h-50"><div class="col border-right">&nbsp;</div><div class=col>&nbsp;</div></div><div class=m-2><span class="badge badge-pill border">&nbsp;</span></div><div class="row h-50"><div class=col>&nbsp;</div><div class=col>&nbsp;</div></div></div><div class="col py-2"><div class=card><div class=card-body><div class="d-flex align-content-start"><div class="mr-2 mb-2"><a href=https://www.rna.uni-jena.de/ target=_blank rel=noopener><img src=./media/icons/brands/fsu.svg width=56px height=56px alt="RNA Bioinformatics & High-Throughput Analysis, University of Jena" loading=lazy></a></div><div><div class="section-subheading card-title exp-title text-muted my-0">PhD Student</div><div class="section-subheading card-title exp-company text-muted my-0"><a href=https://www.rna.uni-jena.de/ target=_blank rel=noopener>RNA Bioinformatics & High-Throughput Analysis, University of Jena</a></div><div class="text-muted exp-meta">Aug 2013 –
Dec 2017
<span class=middot-divider></span>
<span>Jena, Germany</span></div></div></div><div class=card-text>I accomplished my PhD in the research group of Prof. Manja Marz while working on my thesis about &ldquo;The Dark Art of Next-Generation Sequencing: fundamental approaches for genomics, transcriptomics, and differential gene expression&rdquo; (summa cum laude)</div></div></div></div></div></div></div></div></section><section id=contact class="home-section wg-contact"><div class=home-section-bg></div><div class=container><div class=row><div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start"><h1 class=mb-0>Get in touch</h1></div><div class="col-12 col-lg-8"><ul class=fa-ul><li><i class="fa-li fas fa-envelope fa-2x" aria-hidden=true></i>
<span id=person-email><a href=mailto:hoelzer.martin@gmail.com>hoelzer.martin@gmail.com</a></span></li></ul></div></div></div></section></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2024 Martin Hölzer. This work is licensed under <a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank>CC BY SA 4.0</a>.</p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-sa fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Site built with lightly customized version of <a href=https://github.com/wowchemy/starter-hugo-academic target=_blank rel=noopener>Wowchemy Academic</a>.
Code available on <a href=https://github.com/simongravelle/simongravelle.github.io target=_blank rel=noopener>Github</a>.</p><br style=line-height:5px></footer></div></div><script src=./js/vendor-bundle.min.b4708d4364577c16ab7001b265a063a4.js></script><script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script><script id=page-data type=application/json>{"use_headroom":false}</script><script src=./en/js/wowchemy.min.42010733157c11a71adebfe9bae43355.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy
</a><a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=./js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>