<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=generator content="Wowchemy 5.6.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap" media=print onload='this.media="all"'><meta name=author content="Martin Hölzer"><meta name=description content="Personal page"><link rel=alternate hreflang=en-us href=https://hoelzer.github.io/publication.html><meta name=theme-color content="#1565c0"><link rel=stylesheet href=./css/vendor-bundle.min.c7b8d9abd591ba2253ea42747e3ac3f5.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=./css/wowchemy.7d31d7a85d84d3fa1c19b09946b72266.css><link rel=stylesheet href=./css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=./css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><link rel=alternate href=./publication/index.xml type=application/rss+xml title="Martin Hölzer"><link rel=manifest href=./manifest.webmanifest><link rel=icon type=image/png href=./media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=./media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://hoelzer.github.io/publication.html><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@martinhoelzer"><meta property="twitter:creator" content="@martinhoelzer"><meta property="og:site_name" content="Martin Hölzer"><meta property="og:url" content="https://hoelzer.github.io/publication.html"><meta property="og:title" content="Publications | Martin Hölzer"><meta property="og:description" content="Personal page"><meta property="og:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="twitter:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2024-07-05T00:00:00+00:00"><title>Publications | Martin Hölzer</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=./js/wowchemy-init.min.613040fe4f2c0f007b4dcb64404201cb.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><header class=header--fixed><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=./><img src=./media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=./><img src=./media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=./><span></span></a></li><li class=nav-item><a class=nav-link href=./#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=./#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=./#experience><span>Experience</span></a></li><li class=nav-item><a class=nav-link href=./#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://github.com/hoelzer target=_blank rel=noopener aria-label=github><i class="fab fa-github" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span>
</a><a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span>
</a><a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class="universal-wrapper pt-3"><h1>Publications</h1></div><div class=universal-wrapper><div class=row><div class=col-lg-12><div class="form-row mb-4"><div class=col-auto><input type=search class="filter-search form-control form-control-sm" placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off role=textbox spellcheck=false></div><div class=col-auto><select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group=pubtype><option value=*>Type</option><option value=.pubtype-2>Journal article</option></select></div><div class=col-auto><select class="pub-filters form-control form-control-sm" data-filter-group=year><option value=*>Date</option><option value=.year-2024>2024</option><option value=.year-2023>2023</option><option value=.year-2021>2021</option><option value=.year-2020>2020</option><option value=.year-2019>2019</option></select></div></div><div id=container-publications><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2024"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2024-impact-of-reference-design.html>Impact of reference design on estimating SARS-CoV-2 lineage abundances from wastewater sequencing data</a></div><a href=./publication/2024-impact-of-reference-design.html class=summary-link><div class=article-style><strong>Background</strong> Sequencing of SARS-CoV-2 RNA from wastewater samples has emerged as a valuable tool for detecting the presence and relative …</div></a><div class="stream-meta article-metadata"><div><span>Eva Aßmann</span>, <span>Shelesh Agrawal</span>, <span>Laura Orschler</span>, <span>Sindy Böttcher</span>, <span>Susanne Lackner</span>, <span><strong>Martin Hölzer</strong></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/gigascience/giae051 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/rki-mf1/vlq-nf target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/upbqj/ target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2024"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2024-ribap.html>RIBAP: a comprehensive bacterial core genome annotation pipeline for pangenome calculation beyond the species level</a></div><a href=./publication/2024-ribap.html class=summary-link><div class=article-style>Microbial pangenome analysis identifies present or absent genes in prokaryotic genomes. However, current tools are limited when …</div></a><div class="stream-meta article-metadata"><div><span>Kevin Lamkiewicz</span>, <span>Lisa-Marie Barf</span>, <span>Konrad Sachse</span>, <span><strong>Martin Hölzer</strong></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03312-9 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/hoelzer-lab/ribap target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/g52rb target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2024"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2024-pocp-nf.html>POCP-nf: an automatic Nextflow pipeline for calculating the percentage of conserved proteins in bacterial taxonomy</a></div><a href=./publication/2024-pocp-nf.html class=summary-link><div class=article-style><strong>Summary</strong> Sequence technology advancements have led to an exponential increase in bacterial genomes, necessitating robust taxonomic …</div></a><div class="stream-meta article-metadata"><div><span><strong>Martin Hölzer</strong></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/bioinformatics/btae175 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/hoelzer/pocp target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://osf.io/2tzd9 target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2023"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2023-virify.html>VIRify: an integrated detection, annotation and taxonomic classification pipeline using virus-specific protein profile hidden Markov models</a></div><a href=./publication/2023-virify.html class=summary-link><div class=article-style>The study of viral communities has revealed the enormous diversity and impact these biological entities have on various ecosystems. …</div></a><div class="stream-meta article-metadata"><div><span>Guillermo Rangel-Pineros, Alexandre Almeida, Martin Beracochea, Ekaterina Sakharova, Manja Marz, Alejandro Reyes Muñoz, <strong>Martin Hölzer</strong></span>, <span>Robert D Finn</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1371/journal.pcbi.1011422 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/EBI-Metagenomics/emg-viral-pipeline target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.17605/OSF.IO/FBRXY target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2022-skin-microbiome.html>Integrating cultivation and metagenomics for a multi-kingdom view of skin microbiome diversity and functions</a></div><a href=./publication/2022-skin-microbiome.html class=summary-link><div class=article-style>Human skin functions as a physical barrier to foreign pathogen invasion and houses numerous commensals. Shifts in the human skin …</div></a><div class="stream-meta article-metadata"><div><span>Sara Saheb Kashaf</span>, <span>Diana M Proctor</span>, <span>Clay Deming</span>, <span>Paul Saary</span>, <span><strong>Martin Hölzer</strong></span>, <span>Monica E Taylor</span>, <span>Heidi H Kong</span>, <span>Julia A Segre</span>, <span>Alexandre Almeida</span>, <span>Robert D Finn</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.nature.com/articles/s41564-021-01011-w target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/skinmicrobiome/skin_MAGs/releases/tag/v1.0 target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=http://ftp.ebi.ac.uk/pub/databases/metagenomics/genome_sets/skin_microbiome target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Metagenome assemblies</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2021-porecov.html>poreCov - an easy to use, fast, and robust workflow for SARS-CoV-2 genome reconstruction via nanopore sequencing</a></div><a href=./publication/2021-porecov.html class=summary-link><div class=article-style>In response to the SARS-CoV-2 pandemic, a highly increased sequencing effort has been established worldwide to track and trace ongoing …</div></a><div class="stream-meta article-metadata"><div><span>Christian Brandt</span>, <span>Sebastian Krautwurst</span>, <span>Riccardo Spott</span>, <span>Mara Lohde</span>, <span>Mateusz Jundzill</span>, <span>Mike Marquet</span>, <span><strong>Martin Hölzer</strong></span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.3389/fgene.2021.711437 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://github.com/replikation/poreCov target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2020-aquadiva-long-reads.html>Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome‐assembled genomes from a complex aquifer system</a></div><a href=./publication/2020-aquadiva-long-reads.html class=summary-link><div class=article-style>Assembling microbial and viral genomes from metagenomes is a powerful and appealing method to understand structure–function …</div></a><div class="stream-meta article-metadata"><div><span>Will A Overholt</span>, <span><strong>Martin Hölzer</strong></span>, <span>Patricia Geesink</span>, <span>Celia Diezel</span>, <span>Manja Marz</span>, <span>Kirsten Küsel</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1111/1462-2920.15186 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a></div></div><div class=ml-3></div></div></div><div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019"><div class="media stream-item view-compact"><div class=media-body><div class="section-subheading article-title mb-0 mt-0"><a href=./publication/2019-transcriptome-assembly.html>De novo transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers</a></div><a href=./publication/2019-transcriptome-assembly.html class=summary-link><div class=article-style><strong>Background</strong> In recent years, massively parallel complementary DNA sequencing (RNA sequencing [RNA-Seq]) has emerged as a fast, …</div></a><div class="stream-meta article-metadata"><div><span><strong>Martin Hölzer</strong></span>, <span>Manja Marz</span></div></div><div class=btn-links><a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.1093/gigascience/giz039 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://www.rna.uni-jena.de/supplements/assembly/index.html target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Electronic Supplement</a>
<a class="btn btn-outline-primary btn-page-header btn-sm" href=https://doi.org/10.17605/OSF.IO/5ZDX4 target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=ml-3></div></div></div></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2024 Martin Hölzer. This work is licensed under <a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank>CC BY SA 4.0</a>.</p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-sa fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Site built with lightly customized version of <a href=https://github.com/wowchemy/starter-hugo-academic target=_blank rel=noopener>Wowchemy Academic</a>.
Code available on <a href=https://github.com/simongravelle/simongravelle.github.io target=_blank rel=noopener>Github</a>.</p><br style=line-height:5px></footer></div></div><script src=./js/vendor-bundle.min.b4708d4364577c16ab7001b265a063a4.js></script><script src=https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin=anonymous></script><script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script><script id=page-data type=application/json>{"use_headroom":true}</script><script src=./js/wowchemy-headroom.c251366b4128fd5e6b046d4c97a62a51.js type=module></script><script src=./en/js/wowchemy.min.42010733157c11a71adebfe9bae43355.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy
</a><a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=./js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>