---
title: "Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome‐assembled genomes from a complex aquifer system"
date: 2020-08-05
publishDate: 2020-08-05
authors: ["Will A Overholt", "**Martin Hölzer**", "Patricia Geesink", "Celia Diezel", "Manja Marz", "Kirsten Küsel"]
publication_types: ["2"]
abstract: "Assembling microbial and viral genomes from metagenomes is a powerful and appealing method to understand structure–function relationships in complex environments. To compare the recovery of genomes from microorganisms and their viruses from groundwater, we generated shotgun metagenomes with Illumina sequencing accompanied by long reads derived from the Oxford Nanopore Technologies (ONT) sequencing platform. Assembly and metagenome-assembled genome (MAG) metrics for both microbes and viruses were determined from an Illumina-only assembly, ONT-only assembly, and a hybrid assembly approach. The hybrid approach recovered 2× more mid to high-quality MAGs compared to the Illumina-only approach and 4× more than the ONT-only approach. A similar number of viral genomes were reconstructed using the hybrid and ONT methods, and both recovered nearly fourfold more viral genomes than the Illumina-only approach. While yielding fewer MAGs, the ONT-only approach generated MAGs with a high probability of containing rRNA genes, 3× higher than either of the other methods. Of the shared MAGs recovered from each method, the ONT-only approach generated the longest and least fragmented MAGs, while the hybrid approach yielded the most complete. This work provides quantitative data to inform a cost–benefit analysis of the decision to supplement shotgun metagenomic projects with long reads towards the goal of recovering genomes from environmentally abundant groups."
featured: true
publication: "Environmental microbiology, 22(9), 4000-4013"
links:
  - icon_pack: fas
    icon: scroll
    name: Link
    url: 'https://doi.org/10.1111/1462-2920.15186'

---

