---
title: "poreCov - an easy to use, fast, and robust workflow for SARS-CoV-2 genome reconstruction via nanopore sequencing"
date: 2021-07-28
publishDate: 2021-07-28
authors: ["Christian Brandt", "Sebastian Krautwurst", "Riccardo Spott", "Mara Lohde", "Mateusz Jundzill", "Mike Marquet", "**Martin HÃ¶lzer**"]
publication_types: ["2"]
abstract: "In response to the SARS-CoV-2 pandemic, a highly increased sequencing effort has been established worldwide to track and trace ongoing viral evolution. Technologies, such as nanopore sequencing via the ARTIC protocol are used to reliably generate genomes from raw sequencing data as a crucial base for molecular surveillance. However, for many labs that perform SARS-CoV-2 sequencing, bioinformatics is still a major bottleneck, especially if hundreds of samples need to be processed in a recurring fashion. Pipelines developed for short-read data cannot be applied to nanopore data. Therefore, specific long-read tools and parameter settings need to be orchestrated to enable accurate genotyping and robust reference-based genome reconstruction of SARS-CoV-2 genomes from nanopore data. Here we present poreCov, a highly parallel workflow written in Nextflow, using containers to wrap all the tools necessary for a routine SARS-CoV-2 sequencing lab into one program. The ease of installation, combined with concise summary reports that clearly highlight all relevant information, enables rapid and reliable analysis of hundreds of SARS-CoV-2 raw sequence data sets or genomes. poreCov is freely available on GitHub under the GNUv3 license: github.com/replikation/poreCov."
featured: true
publication: "Frontiers in Genetics, 12, 711437"
links:
  - icon_pack: fas
    icon: scroll
    name: Link
    url: 'https://doi.org/10.3389/fgene.2021.711437'
  - icon_pack: fab
    icon: github
    name: Code 
    url: 'https://github.com/replikation/poreCov'

---

