<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=generator content="Wowchemy 5.6.0 for Hugo"><link rel=preconnect href=https://fonts.gstatic.com crossorigin><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Source+Sans+Pro:wght@200;300;400;700&display=swap" media=print onload='this.media="all"'><meta name=author content="Martin Hölzer"><meta name=description content="Time- and cost-saving surveillance of viral pathogens is achieved by tiled sequencing in which a viral genome is amplified in overlapping PCR amplicons and qPCR. However, designing pan-specific primers for viral pathogens with high genomic variability represents a significant challenge. Here, we present a bioinformatics command-line tool, called varVAMP (variable virus amplicons), which addresses this issue. It relies on multiple sequence alignments of highly variable virus sequences and enables degenerate primer design for qPCR or tiled amplicon whole genome sequencing. We demonstrate the utility of varVAMP by designing and evaluating novel pan-specific primer schemes suitable for sequencing the genomes of SARS-CoV-2, Hepatitis E virus, rat Hepatitis E virus, Hepatitis A virus, Borna-disease-virus-1, and Poliovirus using clinical samples. Importantly, we also designed primers on the same input data using the software packages PrimalScheme and Olivar and showed that varVAMP minimizes primer mismatches most efficiently. Finally, we established highly sensitive and specific Poliovirus qPCR assays that could potentially simplify current Poliovirus surveillance. varVAMP is open-source and available through PyPI, UseGalaxy, Bioconda, and https://github.com/jonas-fuchs/varVAMP."><link rel=alternate hreflang=en-us href=https://hoelzer.github.io/publication/2025-varvamp.html><meta name=theme-color content="#1565c0"><link rel=stylesheet href=../css/vendor-bundle.min.c7b8d9abd591ba2253ea42747e3ac3f5.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=../css/wowchemy.7d31d7a85d84d3fa1c19b09946b72266.css><link rel=stylesheet href=../css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=../css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><link rel=manifest href=../manifest.webmanifest><link rel=icon type=image/png href=../media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=../media/icon_hubee73675512ff6f9f19ebbc426d451d2_2177_180x180_fill_lanczos_center_3.png><link rel=canonical href=https://hoelzer.github.io/publication/2025-varvamp.html><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@martinhoelzer"><meta property="twitter:creator" content="@martinhoelzer"><meta property="og:site_name" content="Martin Hölzer"><meta property="og:url" content="https://hoelzer.github.io/publication/2025-varvamp.html"><meta property="og:title" content="varVAMP: degenerate primer design for tiled full genome sequencing and qPCR | Martin Hölzer"><meta property="og:description" content="Time- and cost-saving surveillance of viral pathogens is achieved by tiled sequencing in which a viral genome is amplified in overlapping PCR amplicons and qPCR. However, designing pan-specific primers for viral pathogens with high genomic variability represents a significant challenge. Here, we present a bioinformatics command-line tool, called varVAMP (variable virus amplicons), which addresses this issue. It relies on multiple sequence alignments of highly variable virus sequences and enables degenerate primer design for qPCR or tiled amplicon whole genome sequencing. We demonstrate the utility of varVAMP by designing and evaluating novel pan-specific primer schemes suitable for sequencing the genomes of SARS-CoV-2, Hepatitis E virus, rat Hepatitis E virus, Hepatitis A virus, Borna-disease-virus-1, and Poliovirus using clinical samples. Importantly, we also designed primers on the same input data using the software packages PrimalScheme and Olivar and showed that varVAMP minimizes primer mismatches most efficiently. Finally, we established highly sensitive and specific Poliovirus qPCR assays that could potentially simplify current Poliovirus surveillance. varVAMP is open-source and available through PyPI, UseGalaxy, Bioconda, and https://github.com/jonas-fuchs/varVAMP."><meta property="og:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="twitter:image" content="https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_300x300_fit_lanczos_3.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2025-05-31T00:00:00+00:00"><meta property="article:modified_time" content="2025-05-31T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://hoelzer.github.io/publication/2025-varvamp.html"},"headline":"varVAMP: degenerate primer design for tiled full genome sequencing and qPCR","datePublished":"2025-05-31T00:00:00Z","dateModified":"2025-05-31T00:00:00Z","author":{"@type":"Person","name":"Jonas Fuchs"},"publisher":{"@type":"Organization","name":"Martin Hölzer","logo":{"@type":"ImageObject","url":"https://hoelzer.github.io/media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_192x192_fit_lanczos_3.png"}},"description":"Time- and cost-saving surveillance of viral pathogens is achieved by tiled sequencing in which a viral genome is amplified in overlapping PCR amplicons and qPCR. However, designing pan-specific primers for viral pathogens with high genomic variability represents a significant challenge. Here, we present a bioinformatics command-line tool, called varVAMP (variable virus amplicons), which addresses this issue. It relies on multiple sequence alignments of highly variable virus sequences and enables degenerate primer design for qPCR or tiled amplicon whole genome sequencing. We demonstrate the utility of varVAMP by designing and evaluating novel pan-specific primer schemes suitable for sequencing the genomes of SARS-CoV-2, Hepatitis E virus, rat Hepatitis E virus, Hepatitis A virus, Borna-disease-virus-1, and Poliovirus using clinical samples. Importantly, we also designed primers on the same input data using the software packages PrimalScheme and Olivar and showed that varVAMP minimizes primer mismatches most efficiently. Finally, we established highly sensitive and specific Poliovirus qPCR assays that could potentially simplify current Poliovirus surveillance. varVAMP is open-source and available through PyPI, UseGalaxy, Bioconda, and https://github.com/jonas-fuchs/varVAMP."}</script><title>varVAMP: degenerate primer design for tiled full genome sequencing and qPCR | Martin Hölzer</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=1c014a5343b441b6b68cf4574dd750bc><script src=../js/wowchemy-init.min.613040fe4f2c0f007b4dcb64404201cb.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><header class=header--fixed><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=../><img src=../media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=../><img src=../media/logo_hubee73675512ff6f9f19ebbc426d451d2_2177_0x70_resize_lanczos_3.png alt="Martin Hölzer"></a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=../#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=../#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=../#experience><span>Experience</span></a></li><li class=nav-item><a class=nav-link href=../#contact><span>Contact</span></a></li><li class=nav-item><a class=nav-link href=../gallery.html><span>Scientific Adventures</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class="nav-item d-none d-lg-inline-flex"><a class=nav-link href=https://github.com/hoelzer target=_blank rel=noopener aria-label=github><i class="fab fa-github" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span>
</a><a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span>
</a><a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class=pub><div class="article-container pt-3"><h1>varVAMP: degenerate primer design for tiled full genome sequencing and qPCR</h1><div class=article-metadata><div><span>Jonas Fuchs</span>, <span>Johanna Kleine</span>, <span>Mathias Schemmerer</span>, <span>Julian Kreibich</span>, <span>Wolfgang Maier</span>, <span>Namuun Battur</span>, <span>Thomas Krannich</span>, <span>Somayyeh Sedaghatjoo</span>, <span>Lena Jaki</span>, <span>Anastasija Maks</span>, <span>Christina Boehm</span>, <span>Carina Wilhelm</span>, <span>Jessica Schulze</span>, <span>Christin Mache</span>, <span>Elischa Berger</span>, <span>Jessica Panajotov</span>, <span>Lisa Arnold</span>, <span>Björn Grüning</span>, <span>Markus Bauswein</span>, <span>Sindy Böttcher</span>, <span>Reimar Johne</span>, <span>Jürgen Wenzel</span>, <span><strong>Martin Hölzer</strong></span>, <span>Marcus Panning</span></div><span class=article-date>Posted May, 2025
</span><span class=article-date></span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary btn-page-header" href=https://www.nature.com/articles/s41467-025-60175-9 target=_blank rel=noopener><i class="fas fa-scroll mr-1"></i>Link</a>
<a class="btn btn-outline-primary btn-page-header" href=https://github.com/jonas-fuchs/varVAMP target=_blank rel=noopener><i class="fab fa-github mr-1"></i>Code</a>
<a class="btn btn-outline-primary btn-page-header" href=https://github.com/jonas-fuchs/varVAMP_in_silico_analysis target=_blank rel=noopener><i class="ai ai-open-data mr-1"></i>Open Data</a></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract>Time- and cost-saving surveillance of viral pathogens is achieved by tiled sequencing in which a viral genome is amplified in overlapping PCR amplicons and qPCR. However, designing pan-specific primers for viral pathogens with high genomic variability represents a significant challenge. Here, we present a bioinformatics command-line tool, called varVAMP (variable virus amplicons), which addresses this issue. It relies on multiple sequence alignments of highly variable virus sequences and enables degenerate primer design for qPCR or tiled amplicon whole genome sequencing. We demonstrate the utility of varVAMP by designing and evaluating novel pan-specific primer schemes suitable for sequencing the genomes of SARS-CoV-2, Hepatitis E virus, rat Hepatitis E virus, Hepatitis A virus, Borna-disease-virus-1, and Poliovirus using clinical samples. Importantly, we also designed primers on the same input data using the software packages PrimalScheme and Olivar and showed that varVAMP minimizes primer mismatches most efficiently. Finally, we established highly sensitive and specific Poliovirus qPCR assays that could potentially simplify current Poliovirus surveillance. varVAMP is open-source and available through PyPI, UseGalaxy, Bioconda, and <a href=https://github.com/jonas-fuchs/varVAMP target=_blank rel=noopener>https://github.com/jonas-fuchs/varVAMP</a>.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=../publication.html#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9">Nature Communications, 16 (5067)</div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style></div><script src=../js/jquery-3.6.2.min.js></script><script src=../js/transition.js></script><script src=../js/collapse.js></script><script src=../js/dropdown.js></script><script src=https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js></script><script src=../js/codefolding.js></script><link rel=stylesheet href=../css/codefolding.css><script>$(document).ready(function(){window.initializeCodeFolding("show"==="show")})</script><div class=share-box><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https://hoelzer.github.io/publication/2025-varvamp.html&amp;text=varVAMP:%20degenerate%20primer%20design%20for%20tiled%20full%20genome%20sequencing%20and%20qPCR" target=_blank rel=noopener class=share-btn-twitter aria-label=twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https://hoelzer.github.io/publication/2025-varvamp.html&amp;t=varVAMP:%20degenerate%20primer%20design%20for%20tiled%20full%20genome%20sequencing%20and%20qPCR" target=_blank rel=noopener class=share-btn-facebook aria-label=facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=varVAMP:%20degenerate%20primer%20design%20for%20tiled%20full%20genome%20sequencing%20and%20qPCR&amp;body=https://hoelzer.github.io/publication/2025-varvamp.html" target=_blank rel=noopener class=share-btn-email aria-label=envelope><i class="fas fa-envelope"></i></a></li><li><a href="https://www.tumblr.com/widgets/share/tool?canonicalUrl=https://hoelzer.github.io/publication/2025-varvamp.html&amp;title=varVAMP:%20degenerate%20primer%20design%20for%20tiled%20full%20genome%20sequencing%20and%20qPCR" target=_blank rel=noopener class=share-btn-tumblr aria-label=tumblr><i class="fab fa-tumblr"></i></a></li></ul></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2025 Martin Hölzer. This work is licensed under <a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank>CC BY SA 4.0</a>.</p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-sa/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-sa fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Site built with lightly customized version of <a href=https://github.com/wowchemy/starter-hugo-academic target=_blank rel=noopener>Wowchemy Academic</a>.
Code available on <a href=https://github.com/simongravelle/simongravelle.github.io target=_blank rel=noopener>Github</a>.</p><br style=line-height:5px></footer></div></div><script src=../js/vendor-bundle.min.b4708d4364577c16ab7001b265a063a4.js></script><script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script><script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script><script id=page-data type=application/json>{"use_headroom":true}</script><script src=../js/wowchemy-headroom.c251366b4128fd5e6b046d4c97a62a51.js type=module></script><script src=../en/js/wowchemy.min.42010733157c11a71adebfe9bae43355.js></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy
</a><a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=../js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script></body></html>